### PR TITLE
ADDED: Admin Login

### PR DESCRIPTION
**Pull request by AKSHAYAPALCHAMY-P**

### CODE REVIEW

File: AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+EmployeeDetailMethods.Save = function()
+{
+    const NAME = INP_EMPLOYEE_NAME.value;
+    const AGE = INP_EMPLOYEE_AGE.value;
+    const ADDRESS = INP_EMPLOYEE_ADDRESS.value;
+    const QUALIFICATION = INP_EMPLOYEE_QUALIFICATION.value;
+    const POSITION = INP_EMPLOYEE_POSITION.value;
+    const SALARY = INP_EMPLOYEE_SALARY.value;
+    const GENDER = INP_EMPLOYEE_GENDER_MALE.checked ? 'Male' : INP_EMPLOYEE_GENDER_FEMALE.checked ? 'Female' : '';
```
Comment (ManiRaj-021199) [inline AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js]: const GENDER = INP_EMPLOYEE_GENDER_MALE.checked ? 'Male' : 'Female';

Can we do like this? It's either male or female right?

File: AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+
+    if(isNaN(AGE) || isNaN(SALARY))
+    {
+        alert(AUTHENTICATION_MESSAGES.NeumericValues);
+        return;
+    }
+
+    const employeeDetails = {
+        "Employee Name": NAME,
```
Comment (ManiRaj-021199) [inline AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js]: Do not hardcode the values..

const employeeDetails = { };
employeeDetails[INP_EMPLOYEE_NAME.id] = NAME;
// Other Properties...

Try this...

File: AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+        "Employee Salary": SALARY,
+        "Gender": GENDER,
+        //"Employee Photo": PHOTO_URL
+    };
+
+    let details = JSON.parse(localStorage.getItem("Details")) || [];
+    details.push(employeeDetails);
+    localStorage.setItem("Details", JSON.stringify(details));
+
+    document.getElementById("employeeForm").reset();
```
Comment (ManiRaj-021199) [inline AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js]: document.getElementById("employeeForm")

getElementById should be global constant right?

File: AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+    EmployeeDetailMethods.Display();
+};
+
+EmployeeDetailMethods.Display = function()
+{
+    const details = JSON.parse(localStorage.getItem("Details")) || [];
+    const employeeDetailsContent = document.getElementById("employeeDetailsContent");
```
Comment (ManiRaj-021199) [inline AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js]: document.getElementById("employeeDetailsContent")

Again, It should be global...

File: AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+        "Employee Age": AGE,
+        "Employee Address": ADDRESS,
+        "Employee Qualification": QUALIFICATION,
+        "Employee Position": POSITION,
+        "Employee Salary": SALARY,
+        "Gender": GENDER,
+        //"Employee Photo": PHOTO_URL
+    };
+
+    let details = JSON.parse(localStorage.getItem("Details")) || [];
```
Comment (ManiRaj-021199) [inline AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js]: "Details" ---> Change this to EmployeeDetails

File: AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+    localStorage.setItem("Details", JSON.stringify(details));
+    EmployeeDetailMethods.Display();
+};
+
+EmployeeDetailMethods.Update = function(index)
+{
+    const details = JSON.parse(localStorage.getItem("Details")) || [];
+    const detailToUpdate = details[index];
+
+    INP_EMPLOYEE_NAME.value = detailToUpdate["Employee Name"];
```
Comment (ManiRaj-021199) [inline AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js]: INP_EMPLOYEE_NAME.value = detailToUpdate["Employee Name"];

to

INP_EMPLOYEE_NAME.value = detailToUpdate[INP_EMPLOYEE_NAME.id];

**Review by ManiRaj-021199 — CHANGES_REQUESTED** [review]

File: AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js (hunk 1 above)
Comment (AKSHAYAPALCHAMY-P) [reply AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js]: done

File: AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js (hunk 2 above)
Comment (AKSHAYAPALCHAMY-P) [reply AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js]: done

File: AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js (hunk 5 above)
Comment (AKSHAYAPALCHAMY-P) [reply AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js]: done

File: AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js (hunk 6 above)
Comment (AKSHAYAPALCHAMY-P) [reply AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js]: done

File: AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js (hunk 4 above)
Comment (AKSHAYAPALCHAMY-P) [reply AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js]: done

File: AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js (hunk 5 above)
Comment (AKSHAYAPALCHAMY-P) [reply AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js]: done

**Review by ManiRaj-021199 — COMMENTED** [review]
Looks Good 😊

**Review by ManiRaj-021199 — APPROVED** [review]
Looks Good 😊

**Pull request merged**